test/cases/side-effects/dynamic-reexports/warnings.js

```diff
@@ -1,5 +0,0 @@
-module.exports = [
-	[
-		/export 'default' \(imported as 'Default1'\) was not found in '\.\/direct-export'/
```

Comment:
The error here is important for the test. It need to check that the `default` export is not reexported from a `export *`. Could you verify with `__webpack_exports_info__.default.provided` inside of `direct-export` that it still works as expected.